variants/sparkFun_pro_micro_esp32c3/pins_arduino.h

```diff
@@ -0,0 +1,58 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+#define USB_VID            0x1B4F
+#define USB_PID            0x0035
+#define USB_MANUFACTURER   "SparkFun"
+#define USB_PRODUCT        "SparkFun_Pro_Micro-ESP32C3"
+#define USB_SERIAL         "" // Empty string for MAC adddress
+
+
+#define EXTERNAL_NUM_INTERRUPTS 22
+#define NUM_DIGITAL_PINS        22
+#define NUM_ANALOG_INPUTS       6
+
+#define analogInputToDigitalPin(p)  (((p)<NUM_ANALOG_INPUTS)?(analogChannelToDigitalPin(p)):-1)
+#define digitalPinToInterrupt(p)    (((p)<NUM_DIGITAL_PINS)?(p):-1)
+#define digitalPinHasPWM(p)         (p < EXTERNAL_NUM_INTERRUPTS)
```

Comment:
This part was moved to `Arduino.h` file, as it common for each SoC. Please remove it.

Reply:
Fixed!